**mRNA analysis**    
* Microarray. For Affymetrix expression arrays (HG-U133_Plus_2, HG-U133A, HG_U95Av2 and HuGene-1_0-st) the raw .CEL files (when available) were downloaded from the respective sources and the analyses were completed in R using the [Bioconductor](http://bioconductor.org) suite. The [‘affy’](http://www.bioconductor.org/packages/release/bioc/manuals/affy/man/affy.pdf) package was used for robust multi-array average normalization followed by quantile normalization. For genes with several probe sets, the median of all probes had been chosen. In case the raw data were not available and for other expression array platforms (Illumina beadchip, Agilent-4502A), the available pre-processed expression matrixs were used.            
* RNA-seq. The normalized count reads from the pre-processed data (sequence allignment and transcript abundance estimation) were log2 transformed after adding a 0.5 pseudocount (to avoid infinite value upon log transformation).    

**Copy number**     
* Copy number alterations for the TCGA datasets (obtained through the [GISTIC 2.0](http://www.broadinstitute.org/cgi-bin/cancer/publications/pub_paper.cgi?mode=view&paper_id=216&p=t) algorithm) were downloaded from the [Broad GDAC Firehose](http://gdac.broadinstitute.org).    

**Survival**         
* Kaplan–Meier survival analysis are performed using the [‘survival’](http://cran.r-project.org/web/packages/survival/survival.pdf) package in R. Hazard ratios were determined utilizing the `coxph` function from the ‘survival’ package. To generate HR plot we partially use some code previously describend in [*Cutoff finder*](http://molpath.charite.de/cutoff/index.jsp).    

**Mutations**  
* Mutation data (identified with the MutSig2CV algorithm) are dinamically downloaded from the cBio Portal using the [‘cgdsr’](https://cran.r-project.org/web/packages/cgdsr) package.

